libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -618,8 +618,16 @@ void AP_Mount_Backend::set_rctargeting_on_rcinput_change()
     const int16_t pitch_in = (pitch_ch == nullptr) ? 0 : pitch_ch->get_radio_in();
     const int16_t yaw_in = (yaw_ch == nullptr) ? 0 : yaw_ch->get_radio_in();
 
+    if (!last_rc_input.initialised)
+    {
+            // The first time through, initial RC inputs should be set, but not used
+            last_rc_input.initialised = true;
+            last_rc_input.roll_in = roll_in;
+            last_rc_input.pitch_in = pitch_in;
+            last_rc_input.yaw_in = yaw_in;
+    }
     // if not in RC_TARGETING or RETRACT modes then check for RC change
-    if (get_mode() != MAV_MOUNT_MODE_RC_TARGETING && get_mode() != MAV_MOUNT_MODE_RETRACT) {
+    else if (get_mode() != MAV_MOUNT_MODE_RC_TARGETING && get_mode() != MAV_MOUNT_MODE_RETRACT) {
```

Comment:
Hi, no need for the else.. removing the else will lead to a tiny bit of extra processing where we know that the change in roll, pitch and yaw will be exactly zero but it will only happen once and it makes the code slightly simpler.

Reply:
I see what you mean. I'll test this before I push it again.

Reply:
Fixed, tested - you are right! :-)